pkg/buildimage/chart.go

```diff
@@ -113,12 +113,18 @@ func (c Chart) GetImages() ([]string, error) {
 	delImageSha := func(a string) string {
 		return shaReg.ReplaceAllString(a, "")
 	}
+	// add del -image: and image:
+	delImageReg := regexp.MustCompile(`-{0,1}image:`)
+	delImage := func(str string) string {
+		return delImageReg.ReplaceAllString(str, "")
+	}
 
 	for _, v := range content {
```

Comment:
Use a parse function, and add some unitest for this function:

func parseImages(content) []string {}

Reply:
Need to write a unit test?

Reply:
yes, unit test for this function is important